java/code/src/com/suse/manager/reactor/messaging/test/JobReturnEventMessageActionTest.java

```diff
@@ -341,6 +341,7 @@ public void testPackagesProfileUpdateAllVersions() throws Exception {
                 assertEquals("13.2+git20140911.61c1681", pkg.getEvr().getVersion());
                 assertEquals("12.1", pkg.getEvr().getRelease());
                 assertEquals("x86_64", pkg.getArch().getName());
+                assertEquals(1459866434000L, pkg.getInstallTime().toInstant().toEpochMilli());
```

Comment:
The timestamp comes from https://github.com/uyuni-project/uyuni/blob/master/java/code/src/com/suse/manager/reactor/messaging/test/packages.profileupdate.allversions.json#L53